rest/data/patch.go

```diff
@@ -31,6 +42,10 @@ func (pc *DBPatchConnector) FindPatchesByProject(projectId string, ts time.Time,
 
 // FindPatchById queries the backing database for the patch matching patchId.
 func (pc *DBPatchConnector) FindPatchById(patchId string) (*patch.Patch, error) {
+	if err := validatePatchID(patchId); err != nil {
+		return nil, errors.WithStack(err)
```

Comment:
Does wrapping a `gimlet.ErrorResponse` with `errors.WithStack` return the right thing to the user?

Reply:
yep,